### PR TITLE
feat: Add all subjects option to list consent sessions endpoint

### PR DESCRIPTION
**Pull request by soheiokamoto**
## Proposed changes

This adds `all` subjects option to list consent sessions endpoint: https://www.ory.sh/hydra/docs/reference/api#lists-all-consent-sessions-of-a-subject

This is to allow getting for all subjects explicitly instead of for a single subject.

It follows similar to `all` option for the endpoint to revoke consent sessions for a subject (for a specific client or all clients).

Regenerated swagger based code:
```
make sdk
```

Test passed:
```
make test
```

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

**Comment by aeneasr** [comment]
Thank you, this looks great! Could you please sign the CLA (just two clicks), then we can merge it right away!

**Comment by aeneasr** [comment]
Thank you, this looks great! It looks like the CLA bot is not properly detecting your signature. To fix this, try the following:

```
$ git commit  --amend --author="Author Name <email@address.com>"
```

Ensure that `Author Name` is replaced with your GitHub username (e.g. `aeneasr`) and that the email address is replaced with the email address you have set up in GitHub (e.g. `3372410+aeneasr@users.noreply.github.com`):

```
$ git commit  --amend --author="aeneasr <3372410+aeneasr@users.noreply.github.com>"
```

Once that is done, you can force-push your changes (make sure to push to the correct remote and branch!):

```
$ git push --force <remote> HEAD:<branch>
```

**Comment by aeneasr** [comment]
ping @soheiokamoto :)

**Comment by soheiokamoto** [comment]
@aeneasr Sorry and thanks! I have followed the steps.

**Comment by aeneasr** [comment]
Thank you! :)

File: consent/handler.go [hunk 1]
```diff
@@ -153,13 +153,19 @@ func (h *Handler) DeleteConsentSession(w http.ResponseWriter, r *http.Request, p
 //       500: genericError
 func (h *Handler) GetConsentSessions(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	subject := r.URL.Query().Get("subject")
-	if subject == "" {
-		h.r.Writer().WriteError(w, r, errorsx.WithStack(fosite.ErrInvalidRequest.WithHint(`Query parameter 'subject' is not defined but should have been.`)))
+	allSubjects := r.URL.Query().Get("all") == "true"
```
Comment (aeneasr) [inline consent/handler.go:156]: Sorry, I missed that we still need a test for this. Could you please add one?

**Pull request closed without merging**